src/pretalx.example.cfg

```diff
@@ -8,6 +8,7 @@ static = /var/pretalx/static
 [site]
 debug = False
 url = https://pretalx.mydomain.com
```

Comment:
Please change this to eventyay.mydomain.com

Reply:
updated and will be pushed in next commit